Read both envvars / static error page

mccalluc opened the pull request.

mccalluc committed:
Read both envvars / static error page

scottx611x commented on on_startup.py:
```diff
@@ -135,5 +144,28 @@ def main():
+        )
+        warn(error_str)
+
+        html = '''
```
Awesome! Thanks for doing this!

scottx611x reviewed (approved):
👍

scottx611x reviewed (changes requested):
Just saw the failing build, temp revert of approval

scottx611x commented on on_startup.py:
```diff
@@ -135,5 +144,28 @@ def main():
-    main()
+    try:
+        main()
+    except Exception as e:
```
Failing test seems to be due to the fact  that we're catching something with this except that we weren't before

I remember there being something off with the `django.setup()` within main. Re: [this TODO](https://github.com/refinery-platform/refinery-higlass-docker/pull/15/files#diff-9dfe42fae1cc52c3fbaa4df18f8b4d5aR92).

 Maybe try pulling that out as a call before main and see what happens?

mccalluc committed:
Move error_page inside main and test

mccalluc committed:
Move django.setup out... Scott thinks this will help

mccalluc commented:
Scott and I talked: the error isn't new, but it wasn't being caught this way before. Think it should pass on Travis now, but the underlying issue should not be forgotten.

scottx611x committed:
Remove check for `django.setup()` mock call

`call_command` is already dependent on this being called so its probably overkill
